api/v3/SystemLog.php

```diff
@@ -39,7 +39,7 @@
  * @return array
  */
 function civicrm_api3_system_log_delete($params) {
-  return _civicrm_api3_basic_get(_civicrm_api3_get_BAO(__FUNCTION__), $params, TRUE, 'SystemLog');
+  return _civicrm_api3_basic_delete(_civicrm_api3_get_BAO(__FUNCTION__), $params);
```

Comment:
ah now seeing comment in relevance not just from MM channel can see your point. Yeh i dunno it smells of copy and paste to me